fe/fe-core/src/main/java/com/starrocks/external/iceberg/IcebergUtil.java

```diff
@@ -21,8 +22,17 @@
 import java.util.List;
 import java.util.Map;
 import java.util.Optional;
+import java.util.Properties;
 
 public class IcebergUtil {
+    
+    static {
+        if (Config.iceberg_enable_custom_worker_thread) {
```

Comment:
I think this configure can't work as you expected.
When this block is executed, this value would not be initialized, and this value isn't determined by config file.